scripts/policy/misc/scan.zeek

```diff
@@ -4,6 +4,8 @@
 #            Seth Hall
 #            All the authors of the old scan.bro
 
+@deprecated "Remove in v6.1. Use the external github.com/ncsa/bro-simple-scan package instead (e.g., by installing it via `zkg install ncsa/bro-simple-scan`). The misc/scan.zeek script hasn't been maintained since 2013. Further, the external bro-simple-scan package from NCSA (Justin Azhoff) has become the recommended alternative for TCP scan detection."
```

Comment:
Minor nit: "Azhoff" -> "Azoff" :-)

@JustinAzoff it would be nice to zeekify in the NCSA packages ... is this something you could see doing? Or perhaps someone else over there?

Reply:
The package was zeek'd a while ago, the name is the main thing that needs updating.  I THINK renaming it won't break things for anyone, since github does redirects when you rename a repo.  However I don't have access to the repository anymore, but I can make some inqueries.

Reply:
Thanks, yeah exactly — Github-level repo renaming should work fine. The rest would just be scattered old strings, like `:bro:id:, `bro plugin`, the README content, etc. Really quick changes.

Reply:
> Minor nit: "Azhoff" -> "Azoff" :-)

Uhh.. Fixed now and squashed things together.